scripts/distribution/macOS-package/template/scripts/preinstall

```diff
@@ -22,8 +22,14 @@ function removeServiceIfExists() {
 }
 
 function removeSymlinks(){
+    echo "Removing symlinks:"
+
     removeFileIfExists "/usr/local/bin/concordium-node"
     removeFileIfExists "/usr/local/bin/concordium-node-collector"
+    removeFileIfExists "/Library/LaunchDaemons/software.concordium.mainnet.node.plist"
```

Comment:
1st. this is not a symlink?
2nd. should the configurations be removed?

Reply:
1. These *are* symlinks to actual files that exist in `/Library/Concordium Node/LaunchDaemons/`.
  - macOS will launch any services that exist in `/Library/LaunchDaemons/` on boot. So if you choose to launch the mainnet service on boot, the `postinstall` script will make a symlink from `/Library/LaunchDaemons/software.concordium.mainnet.node.plist -> /Library/Concordium Node/LaunchDaemons/software.concordium.mainnet.node.plist`
2. We could let the `postinstall` script remove or add them based on the configuration options you chose in the installer. But I thought it would be cleaner to simply remove all of them in `preinstall` and then create the necessary ones in `postinstall`.